test.js

```diff
@@ -1,8 +1,7 @@
-'use strict';
-var test = require('ava');
-var camelcaseKeys = require('./');
+import test from 'ava';
+import camelcaseKeys from './';
 
-test(function (t) {
-	t.assert(camelcaseKeys({'foo-bar': true}).fooBar);
+test('camelcase keys', t => {
```

Comment:
Agreed. It used to be like that, but was regressed: https://github.com/sindresorhus/ava/issues/112